logger.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+
+class IgnoreFilter(logging.Filter):
+    """
+    Customized Log filter that ignores a given message text.
+    """
+
+    def __init__(self, ignored_message: str, name: str = "") -> None:
+        super().__init__(name)
+        self.ignored_message = ignored_message
+        self.enabled = True
+
+    def filter(self, record: logging.LogRecord) -> bool:
+        if not self.enabled:
+            return True
+        return self.ignored_message not in record.getMessage()
+
+    def set_ignored_message(self, message: str) -> None:
+        """Set a new message pattern to ignore."""
+        self.ignored_message = message
+
+    def toggle_filter(self, enable: bool) -> None:
+        """Enable or disable the filter."""
+        self.enabled = enable
+
+    def get_ignored_message(self) -> str:
+        """Get the current ignored message pattern."""
+        return self.ignored_message
+
+
+def set_logger(name: str) -> logging.Logger:
+    logging.basicConfig(level=logging.INFO)
+    logging.getLogger("near_lake_framework").setLevel(logging.INFO)
+    missing_shard_filter = IgnoreFilter("doesn't exist")
```

Comment:
Is the missing shard data something that gets backfilled in future data from near lake or skipped?

Reply:
My understanding is that there was no data on the shared corresponding to the block. I think this filter can be removed if we downgrade this log to debug and set the lake log level to info.

Reply:
Created a PR here: https://github.com/frolvanya/near-lake-framework-py/pull/16

We can leave this open and wait for the next version bump (if its approved) - then remove the whole filter class.